fdbserver/DataDistribution.actor.cpp

```diff
@@ -765,6 +767,12 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 				} else {
 					self->medianAvailableSpace = SERVER_KNOBS->MIN_AVAILABLE_SPACE_RATIO;
 				}
+				if (self->medianAvailableSpace < SERVER_KNOBS->TARGET_AVAILABLE_SPACE_RATIO) {
+					TraceEvent(SevWarn, "DDTeamMedianAvailableSpaceTooSmall")
```

Comment:
can we add `distributorId` and `primary` field in the trace event as well?
All DD related events should have these two information, because they are useful in (1) identifying which side (primary or remote) is having issues and (2) when DD reboots, we know if information is from old or new DD.

Reply:
Good catch. Then we need to audit DD's trace log. There are lots of log lines that do not have those two fields.

Reply:
yes, please do that in a separate PR.